scripts/ci_check_no_file_changes.sh

```diff
@@ -27,7 +27,7 @@ if [[ -z ${PR_NUMBER} ]]; then
   exit 1
 fi
 
-URL="https://api.github.com/repos/${GITHUB_REPO}/pulls/${PR_NUMBER}/files"
+URL="https://api.github.com/repos/${GITHUB_REPO}/pulls/${PR_NUMBER}/files?per_page=1000"
```

Comment:
Looks like I'm late to the party again. The maximum `per_page` size is `100` according to the document. To obtain more than 100 items, there is a convenient [`pagination` helper](https://octokit.github.io/rest.js/v16#pagination) in [GitHub script](https://github.com/actions/github-script)

[This PR](https://github.com/apache/superset/pull/13498/files#diff-03633ab82f2a106ba87da31b24dcac2432fbda5f3b4bf79901d95bf4d45fc413R31) has an example.

Reply:
I was actually intending to make a cleaner solution, this is to unblock immediate PRs suffering from this. I'll fix this properly this week.